### PR TITLE
added .property file and loader

### PR DESCRIPTION
**Pull request by sturcanu**

### CODE REVIEW

File: src/main/java/com/endava/soa4/PropertyLoader.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.endava.soa4;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyLoader {
+    public static void main(String[] args) throws IOException {
```
Comment (venache1) [inline src/main/java/com/endava/soa4/PropertyLoader.java]: Please make it as a simple methot instead ov publis static void main. And you will call it from DriverManager class.

File: src/main/resources/config/config.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+url= http://automationpractice.com/index.php
```
Comment (venache1) [inline src/main/resources/config/config.properties]: rename it to a more suggestive variable e.g. **homeUrl**

**Comment by venache1** [comment]
Please add to commit message **Closes #10 #9**

File: src/main/java/com/endava/soa4/PropertyLoader.java (hunk 1 above)
Comment (sturcanu) [reply src/main/java/com/endava/soa4/PropertyLoader.java]: Done

File: src/main/resources/config/config.properties (hunk 2 above)
Comment (sturcanu) [reply src/main/resources/config/config.properties]: Done

**Review by venache1 — APPROVED** [review]

File: src/main/java/com/endava/soa4/PropertyLoader.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import java.util.Properties;
+
+public class PropertyLoader {
+    public static String getProperties(String key) throws IOException {
+        Properties prop = new Properties();
+        InputStream ip = ClassLoader.getSystemResourceAsStream("config/config.properties");
+        prop.load(ip);
+        return prop.getProperty(key);
+    }
+}
```
Comment (venache1) [inline src/main/java/com/endava/soa4/PropertyLoader.java]: Move this class in another package called **utils** under com.endava.soa4 package

File: src/main/java/com/endava/soa4/PropertyLoader.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package com.endava.soa4;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyLoader {
+    public static String getProperties(String key) throws IOException {
+        Properties prop = new Properties();
+        InputStream ip = ClassLoader.getSystemResourceAsStream("config/config.properties");
```
Comment (VitalinaPetrenco) [inline src/main/java/com/endava/soa4/PropertyLoader.java]: Please close inputStream after you finished to use it.

File: src/main/java/com/endava/soa4/PropertyLoader.java (hunk 3 above)
Comment (sturcanu) [reply src/main/java/com/endava/soa4/PropertyLoader.java]: Done

File: src/main/java/com/endava/soa4/PropertyLoader.java (hunk 4 above)
Comment (sturcanu) [reply src/main/java/com/endava/soa4/PropertyLoader.java]: changed it to try-with-resources

**Pull request merged**